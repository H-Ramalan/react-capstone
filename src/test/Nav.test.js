import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Navbar from '../components/Nav';

describe('Navbar Component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Navbar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
