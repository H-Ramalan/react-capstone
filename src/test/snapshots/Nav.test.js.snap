// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component matches snapshot 1`] = `
<DocumentFragment>
  <nav>
  
    
    <h1>
      Countries!
    </h1>
    
     
       
    
  </nav>
</DocumentFragment>
`;